runtime/src/tokio/runtime.rs

```diff
@@ -168,6 +166,79 @@ impl Default for Config {
             maximum_buffer_size: 2 * 1024 * 1024, // 2 MB
         }
     }
+
+    /// Number of threads to use for handling async tasks.
+    pub fn with_worker_threads(mut self, n: usize) -> Self {
+        self.worker_threads = n;
+        self
+    }
+    /// Maximum number of threads to use for blocking tasks.
+    pub fn with_max_blocking_threads(mut self, n: usize) -> Self {
+        self.max_blocking_threads = n;
+        self
+    }
+    /// Whether or not to catch panics.
+    pub fn with_catch_panics(mut self, b: bool) -> Self {
+        self.catch_panics = b;
+        self
+    }
+    /// Duration after which to close the connection if no message is read.
+    pub fn with_read_timeout(mut self, d: Duration) -> Self {
+        self.read_timeout = d;
+        self
+    }
+    /// Duration after which to close the connection if a message cannot be written.
+    pub fn with_write_timeout(mut self, d: Duration) -> Self {
+        self.write_timeout = d;
+        self
+    }
+    /// Whether or not to disable Nagle's algorithm.
+    pub fn with_tcp_nodelay(mut self, n: Option<bool>) -> Self {
+        self.tcp_nodelay = n;
+        self
+    }
+    /// Base directory for all storage operations.
+    pub fn with_storage_directory(mut self, p: impl Into<PathBuf>) -> Self {
+        self.storage_directory = p.into();
+        self
+    }
+    /// Maximum buffer size for operations on blobs.
+    pub fn with_maximum_buffer_size(mut self, n: usize) -> Self {
+        self.maximum_buffer_size = n;
+        self
+    }
+
+    // Getters
```

Comment:
Should these have the same comments?

Reply:
Your call. Honestly, I'd rather not. I was actually considering going in the opposite direction and having the comment on the setters as `/// See [Config]`.

Personally I'm not a fan of duplicating comments because it makes maintaining more annoying. That said, it's nice to have things be documented "in place,: so up to you.

Reply:
Your call. Honestly, I'd rather not. I was actually considering going in the opposite direction and having the comment on the setters as `/// See [Config]`.

Personally I'm not a fan of duplicating comments because it makes maintaining more annoying. That said, it's nice to have things be documented "in place,: so up to you.

Reply:
Your call. Honestly, I'd rather not. I was actually considering going in the opposite direction and having the comment on the setters as `/// See [Config]`.

Personally I'm not a fan of duplicating comments because it makes maintaining more annoying. That said, it's nice to have things be documented "in place,: so up to you.

Reply:
Oooooo I like the `See [Config]`. Would be down to do the same for the setters tbh.